File: src/targets/php/http2/fixtures/multipart-form-data-no-params.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

<<<<<<< HEAD
$request->setRequestUrl('https://httpbin.org/anything');
=======
$request->setRequestUrl('http://mockbin.com/har');
>>>>>>> upstream/master
$request->setRequestMethod('POST');
$request->setHeaders([
  'Content-Type' => 'multipart/form-data'
]);

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: src/targets/csharp/restsharp/fixtures/multipart-form-data-no-params.cs
<<<<<<< HEAD
var client = new RestClient("https://httpbin.org/anything");
=======
var client = new RestClient("http://mockbin.com/har");
>>>>>>> upstream/master
var request = new RestRequest(Method.POST);
request.AddHeader("Content-Type", "multipart/form-data");
IRestResponse response = client.Execute(request);